### PR TITLE
add tags filter to djs page

### PR DESCRIPTION
**Pull request by dudethatbe**

### CODE REVIEW

**Comment by dudethatbe** [comment]
This is technically working, but there's some issue with the multi-select component isn't showing the selected filter until you click on it twice. IDK maybe I didn't use it right either 😅
demonstrating the lemon badge filter works but for some reason the selected filter doesn't persist until its second attempt

File: app/components/djs/search.hbs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<PowerSelectMultiple
+  @options={{this.badges}}
+  @selected={{this.selectedBadges}}
+  @allowClear={{true}}
+  @placeholder="Select some badges..."
```
Comment (mcfiredrill) [inline app/components/djs/search.hbs:5]: Maybe `Filter by badge` ? And let's get this in i18n

**Comment by dudethatbe** [comment]
lets merge #1310  first and then I can add the pineapple badge as an option

**Comment by mcfiredrill** [comment]
> lets merge https://github.com/datafruits/datafruits/pull/1310 first and then I can add the pineapple badge as an option
donezo

File: app/components/djs/search.hbs (hunk 1 above)
Comment (dudethatbe) [reply app/components/djs/search.hbs]: k boss should be good now

**Pull request merged**